tokio/src/runtime/driver.rs

```diff
@@ -80,6 +80,18 @@ impl Handle {
 
         self.io.unpark();
     }
+
+    cfg_time! {
+        /// Returns a reference to the time driver handle.
+        ///
+        /// Panics if no time driver is present
```

Comment:
```suggestion
        /// Panics if no time driver is present.
```

Reply:
This is already tested [here](https://github.com/tokio-rs/tokio/blob/master/tokio/tests/time_panic.rs#L73)